reconcile/cli.py

```diff
@@ -584,6 +585,22 @@ def integration(
     ctx.obj["dump_schemas_file"] = dump_schemas_file
 
 
+@integration.result_callback()
+def exit_integration(
+    ctx,
+    configfile,
+    dry_run,
+    early_exit_compare_sha,
+    check_only_affected_shards,
+    validate_schemas,
+    dump_schemas_file,
+    log_level,
+    gql_sha_url,
+    gql_url_print,
+):
+    GqlApiSingleton().close()
```

Comment:
can directly call `close` since it's a class method
```python
GqlApiSingleton.close()
```

Reply:
Will adjust